application/extensions/admin/survey/ListSurveysWidget/views/listSurveys.php

```diff
@@ -68,7 +68,7 @@
                 [
                     'header'            => gT('Title'),
                     'name'              => 'title',
-                    'value'             => '$data->defaultlanguage->surveyls_title',
```

Comment:
Huh? don't think that belongs in here

Reply:
@c-schmitz if I create a survey with Hausa as its base language and then listing the surveys, then without the fix above I have:

![image](https://github.com/LimeSurvey/LimeSurvey/assets/2805488/0f9ba69d-2850-42ec-b89f-7a3c155d2c1a)

whereas, if the fix above is in place, then:

![image](https://github.com/LimeSurvey/LimeSurvey/assets/2805488/6f8746ff-22e6-4fcb-a4fa-a08f01982645)

If you still prefer this adjustment not to exist, then I will remove it.

Reply:
That is because you have faulty surveys in your database where the entries in surveys_languagesettings are missing.  Sure, this will prevent the notice but strictly speaking still does not belong  to 'Implement support for Hausa language'

Reply:
Thanks for the insight